src/schemas/replicated.json

```diff
@@ -1,1733 +1,657 @@
 {
-  "$schema": "http://json-schema.org/schema#",
```

Comment:
This file is unused, we transpiled it to typescript in `parsed.ts` to make it easier to include in the module.

This file can be deleted if you want to.